docs/console-virtual-terminal-sequences.md

```diff
@@ -72,7 +72,7 @@ Cursor movement will be bounded by the current viewport into the buffer. Scrolli
 | ESC \[ &lt;n&gt; A | CUU | Cursor Up | Cursor up by &lt;n&gt; |
 | ESC \[ &lt;n&gt; B | CUD | Cursor Down | Cursor down by &lt;n&gt; |
 | ESC \[ &lt;n&gt; C | CUF | Cursor Forward | Cursor forward (Right) by &lt;n&gt; |
-| ESC \[ &lt;n&gt; D | CUB | Cursor Backward | Cursor backward (Left) by &lt;n&gt; |
+| ESC \[ &lt;n&gt; D | IND | Cursor Down | Cursor down one line from current position in the same column |
```

Comment:
I've updated the notes here: https://github.com/MicrosoftDocs/Console-Docs/issues/94#issuecomment-1117609251

We do not need to make this change, because `ESC [ D` is correct (`CUB`)

Reply:
Reverted